### DISCUSSION TITLE
Radiation Containtment

### DISCUSSION THREAD

**Discussion by solundev** [HbmMods/Hbm-s-Nuclear-Tech-GIT#502] [Q&A] [answered]
So, is there a way to eficiently contain radiation generated from RBMK reactor and it's gas produced? (not exploded)

**Comment by ncr-t** [comment, marked as answer]
Put lids on top of your RBMK, and encase it in concrete.

**Reply by solundev** [reply]
if it's not trouble for you my friend, can you DM me on Discord? it's Solaris#0006
we could talk better there and freely
and thank you for the tip ^^

**Reply by solundev** [reply]
also, which would be the lids?

**Reply by Froooz** [reply]
RBMK cover panel

**Reply by solundev** [reply]
thanks, so i use it on the fuel rods directly?

**Comment by SoB1nary** [comment]
on *all* the rbmk columns

**Reply by solundev** [reply]
thx, also the reactor is underground and all the facility is encased in concrete, should it work?
![image](https://user-images.githubusercontent.com/82925878/157999142-173da0de-afd8-47fd-ba80-a6cd2ee7768d.png)

**Reply by solundev** [reply]
@-@
![image](https://user-images.githubusercontent.com/82925878/157999367-d1141adb-8bd3-4261-aeae-4cfa9f559a9c.png)

**Reply by SoB1nary** [reply]
how about you show your rbmk

**Reply by solundev** [reply]
of course there it is
![image](https://user-images.githubusercontent.com/82925878/158000052-7e6d55a5-504e-4505-a2f9-1b6ebc559683.png)

**Reply by SoB1nary** [reply]
ah, "encasing the reactor in concrete" means covering it from the sides

**Reply by solundev** [reply]
so i basicly fill with concrete where are air empty spaces? arround the reactor

**Reply by burningwater202** [reply]
I think you can get away with having a single layer of concrete directly on the sides of the reactor, make sure no side is left uncovered

**Reply by solundev** [reply]
okay, will do so and return to tell the results =)

**Reply by solundev** [reply]
it is finally done my friend! thank you so much you all 100% fresh air now <3
